<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTitleBar</class>
 <widget class="QWidget" name="MyTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>25</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>25</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lenis_win_title_lb">
     <property name="text">
      <string>标题</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="lenis_win_wid" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="MyClickedBtn" name="lenis_win_min_btn">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyClickedBtn" name="lenis_win_max_btn">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyClickedBtn" name="lenis_win_close_btn">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyClickedBtn</class>
   <extends>QPushButton</extends>
   <header>myclickedbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
